src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -141,6 +141,61 @@ string GetCompilerDir(const char* compiler) {
   return compiler_dir;
 }
 
+string Trim(const string& cmd) {
```

Comment:
absl has a StripWhitespace function.

Reply:
This trims any non-alphabets in the beginning and end of string - a little different from abls's StripWhitespace